docs/how-to.md

```diff
@@ -60,6 +60,15 @@ background processes, e.g. those for querying resources.
     It is currently not possible to view resources for multiple/all
     namespaces. See [issue #72][issue #72] for details.
 
+### Selecting views
+| Keybinding | Interactive function                   |
+|:-----------|:---------------------------------------|
+| `v`        | `M-x kubernetes-overview-select-views` |
+
+!!! info
+
+    Through `kubernetes-overview-select-views` (`v`) a list of views can be selected. The selected views will be rendered above one another in the overview buffer. Additional (custom) views can be defined through the customizable `kubernetes-overview-custom-views-alist` variable.
```

Comment:
If you could provide screenshots here I think that'd make this section super helpful.

```suggestion
Through `kubernetes-overview-select-views` (`v`) a list of views can be selected. The selected views will be rendered above one another in the overview buffer.
```

Let's move the information about defining custom views into a new `Interface Customization` section elsewhere in this page.